Automattic/wp-calypso
# Jetpack License Portal: Update issue license screen to new design

#### Changes proposed in this Pull Request

* This PR simply updates the design of the license issue screen to be in line with what's expected for the license attaching project.

#### Testing instructions

* Ensure the new issue license screen is applied.
* Ensure that you can issue licenses as usual via the new design.
